build.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 yarn install --immutable
-yarn workspaces foreach --exclude "@kaoto/step-extension-repository" --parallel --verbose run clean
-yarn workspaces foreach --exclude "@kaoto/step-extension-repository" --parallel --verbose run build
+yarn workspaces foreach --exclude "@kaoto/step-extension-repository" --exclude common --parallel --verbose run clean
+yarn workspaces foreach --exclude "@kaoto/step-extension-repository" --exclude common --parallel --verbose run build
```

Comment:
This, on the other hand will require ``--topological-dev` to build the dependencies in the right other:
```suggestion
yarn workspaces foreach --exclude "@kaoto/step-extension-repository" --topological-dev --parallel --verbose run build
```

Reply:
From the docs https://yarnpkg.com/cli/workspaces/foreach:

> `--topological-dev`: Run the command after all workspaces it depends on (regular + dev) have finished